src/configuration.h

```diff
@@ -84,14 +84,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define GPS_SERIAL_NUM 1
 #define GPS_BAUDRATE 9600
 
-#if defined(T_BEAM_V10)
 #define GPS_RX_PIN 34
```

Comment:
Is the intent that we'll always try GPS, even on Heltec?

Reply:
yep - someone asked for allowing gps access on a heltecish board (he has an external gps).  Also, it will make the two platforms less different.